dora/core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -2560,6 +2560,20 @@ public static String getSyncMetricName(long mountId) {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey CLIENT_WORKER_READ_UFS_FALLBACK =
+      new Builder("Client.WorkerReadUfsFallback")
+          .setDescription("Total number of data read fallback to UFS "
+              + "after encountering error when reading from worker.")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey CLIENT_WORKER_READ_UFS_FALLBACK_BYTES =
+      new Builder("Client.WorkerReadUfsFallbackBytes")
```

Comment:
The name is confusing: it tempts me to think this fallback happens on the worker.

```suggestion
      new Builder("Client.UfsFallbackReadBytes")
```

Reply:
+1